pubspec.lock

```diff
@@ -508,11 +508,11 @@ packages:
     dependency: transitive
     description:
       path: "packages/flutter_nekoton_bridge"
-      ref: "flutter_nekoton_bridge-v1.8.0-dev.12"
-      resolved-ref: "5319f6e277d35019c0281ff5f75543748b6c3654"
+      ref: "flutter_nekoton_bridge-v1.8.0-dev.13"
```

Comment:
We have bridge 14 version

Reply:
This is transitive dependency, we shouldn't fix it manually.